### ISSUE TITLE
Missing Copyright

### ISSUE THREAD

**Issue by mbanck** [libmbd/libmbd#55]
The code is cleary under the MPL, but nowhere is it stated who holds the copyright, unless I am missing something?

**Comment by jhrmnn** [comment]
It isn’t indeed. I don’t think a copyright notice is necessary.

**Comment by mbanck** [comment]
In effect, each line of code is under the copyright of the person that wrote it, so it just makes it very tedious and cumbersome to determine the copyright for an outsider, based on git blame. In the current case, that has been pretty clear as most/all commits so far were by you. So a general copyright notice would be appreciated.

This is not an academical question, I needed to package libmbd for Debian/Ubuntu as it is used by newer versions of QE. I went with `Copyright 2015-2023 Jan Hermann <dev@jan.hermann.name>`, but https://github.com/libmbd/libmbd/commit/ac4e807dbd97ec569f02b41b3ae1403097b187a1 now makes that moot in practice. On the other hand, now that libmbd has been accepted in Debian and went through copyright review, this is no longer a showstopper for us.
